react-stopwatch/my-app/src/Timewatch.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import './Timewatch.css';
+import { FaPlay, FaPause } from 'react-icons/fa';
+
+export function Timewatch() {
+  const [count, setCount] = useState(0);
+  const [intervalId, setIntervalId] = useState<NodeJS.Timeout>();
+  const running = intervalId !== undefined;
+
+  function handlePlay() {
+    const id = setInterval(() => setCount((count) => count + 1), 1000);
+    setIntervalId(id);
+  }
+  function handlePause() {
+    clearInterval(intervalId);
+    setIntervalId(undefined);
+  }
+  function clearReset() {
+    if (intervalId) {
+      clearInterval(intervalId);
+      setIntervalId(undefined);
+    }
+    setCount(0);
+  }
+
+  return (
+    <div className="stopwatch">
+      <div className="watch-face" onClick={clearReset}>
+        <span> {count}</span>
+      </div>
+      {!running ? (
+        <button
+          type="submit"
```

Comment:
You don't want `type="submit"` on the button. That makes it a form submit button. If you just have a button sitting there without a form you want `type="button"`.